packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -251,8 +251,7 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
     }
   }
 
-  private _getWeeks(navigatedDate: Date, selectedDate: Date, todayDate: Date): IDayInfo[][] {
-    let { firstDayOfWeek, dateRangeType } = this.props;
+  private _getWeeks(navigatedDate: Date, selectedDate: Date, dateRangeType: DateRangeType, firstDayOfWeek: DayOfWeek, todayDate: Date): IDayInfo[][] {
```

Comment:
Why not just pass in the props object here, rather than 5 different arguments (all of which are sourced directly from a props object)?